src/Tests/Tests/ClientConcepts/Troubleshooting/DebugInformation.doc.cs

```diff
@@ -41,6 +41,41 @@ public void DefaultDebug()
             );
 
             response.DebugInformation.Should().Contain("Valid NEST response");
+        }
+		//hide
+        [U] public void PasswordIsNotExposedInDebugInformation()
```

Comment:
Notice we have spaces here, since `*.doc.cs` have `formatter off` this was not fixed as part of our recent automated code cleanup commits. We really need a `order file off` mechanism but I have not found a way to do that in file comments or editorconfig setting. Will tweet @ jetBrainsRider to see if I missed anything and will fix this on my plane ride home for the active integration branches manually.

Reply:
This should now be fixed on `master" and this PR